### ISSUE TITLE
can't build in ~/.local in Fedora Silverblue

### ISSUE THREAD

**Issue by tazihad** [Vanilla-OS/apx#166]
Trying to install `apx` in Silverblue locally. But it is asking for sudo access.  
```
git clone --recursive https://github.com/Vanilla-OS/apx.git
cd apx
make build PREFIX=$HOME/.local
make install PREFIX=$HOME/.local
```
error
```
install -Dm755 apx //home/zihad/.local/bin/apx
mkdir -p //etc/apx
mkdir: cannot create directory ‘//etc/apx’: Permission denied
make: *** [Makefile:12: install] Error 1
```

any solution? Why it needs access to /etc/apx. Can't it be build in `~/.local`?

**Comment by dch82** [comment]
Try setting `DESTDIR` to something like `$HOME`

**Comment by axtloss** [comment]
The issue should be fixed in #168, for now you can edit the Makefile and change the command `mkdir -p ${DESTDIR}/etc/apx` to `sudo mkdir -p ${DESTDIR}/etc/apx`

**Comment by spaceguybob** [comment]
interesting, i got a similar error
